e2e/__tests__/forceExit.test.js

```diff
@@ -28,18 +28,17 @@ test('exits the process after test are done but before timers complete', () => {
     'package.json': JSON.stringify({jest: {testEnvironment: 'node'}}),
   });
 
-  let stdout;
```

Comment:
I think relying on the frozen console to "redirect" to `stderr` in this test is a bit obscure.
Would be better to have the timer `console.error` right away (or if that doesn't work for some reason, write a file to `DIR`)

Reply:
it just expects it to be somewhere in the output, it doesn't care which stream it's in. I can do `stderr + stdout` if you think that's clearer?

Reply:
The problem is that this test, although it tests `forceExit`, would fail if the `freezeConsole` mechanic was removed. Reason: `setTimeout(() => console.log('TIMER_DONE'), 500);` prints to stdout, but `expect(stderr).toMatch(/TIMER_DONE/);` expects it printed to stderr. The only reason that this works is because of the `freezeConsole` mechanic, so this test implicitly also tests `freezeConsole` as well.

Does that make sense?

Reply:
Well I guess that works too :D